features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx

```diff
@@ -164,8 +164,8 @@ class Details extends Component {
                     <div className={classes.contentDown}>
                         <Switch>
                             <Redirect exact from='/applications/:applicationId' to={redirect_url} />
-                            <Route path='/applications/:applicationId/productionkeys' render={() => <TokenManager keyType='PRODUCTION' selectedApp={{ appId: this.state.application.applicationId, label: this.state.application.name }} />} />
-                            <Route path='/applications/:applicationId/sandBoxkeys' render={() => <TokenManager keyType='SANDBOX' selectedApp={{ appId: this.state.application.applicationId, label: this.state.application.name }} />} />
+                            <Route path='/applications/:applicationId/productionkeys' component={() => <TokenManager keyType='PRODUCTION' selectedApp={{ appId: this.state.application.applicationId, label: this.state.application.name }} />} />
```

Comment:
Shall we destruct the state variables in the `render` method body and use the destruct variable names here.

i:e 

```javascript
const { application } = this.state;
if(application)
const { applicationId, applicationId } = application;
```

Reply:
destruct done, do not need the if condition as it is checked before the render is called

Reply:
Ack :+1: